bubblesub/ui/video.py

```diff
@@ -723,3 +723,6 @@ def __init__(self, api: Api, parent: QtWidgets.QWidget) -> None:
         layout.setContentsMargins(0, 0, 0, 0)
         layout.addLayout(left_layout)
         layout.addLayout(right_layout)
+
+    def set_visibility(self, show_volume_control: bool) -> None:
+        self._volume_control.setVisible(show_volume_control)
```

Comment:
Let's create a signal in the view manager that the layout has changed and listen to it here